src/routes/feedback/components/FeedbackBox.js

```diff
@@ -148,47 +152,45 @@ export class FeedbackBox extends React.Component<PropsType, StateType> {
       }
       // We don't want to differ between the content of categories, extras and events for the user, but we want to know
       // from which route the feedback was sent
-      options.push(this.getFeedbackOption(`${t('contentOfCity')} ${cityTitle}`, feedbackType))
+      options.push(new FeedbackDropdownItem(`${t('contentOfCity')} ${cityTitle}`, feedbackType))
+      this.getExtrasFeedbackOptions().forEach(option => options.push(option))
```

Comment:
If `getExrasFeedbackOptions()` returns an array, this should also work:   
`options = options.concat(this.getExtrasFeedbackOptions())`

Reply:
Doesn't work for some reason, I don't know why.